documentation/synthetic_tokens/tutorials/creating_from_sponsor_cli.md

```diff
@@ -0,0 +1,207 @@
+# UMA Sponsor CLI Tool
+
+The Sponsor CLI tool lets you interact with a synthetic token contract.
+In this tutorial, you’ll be deploying a synthetic token contract to a local testnet and interacting with it via the Sponsor CLI tool.
+
+It will let you be the “Token Sponsor,” which lets you:
+
+- Deposit collateral (ETH) and borrow synthetic tokens
+- Manage your position’s collateral (deposit/withdraw)
```

Comment:
nit: is the `(deposit/withdraw)` redundant if you describe Deposit and Withdraw features directly below?

Reply:
Good catch - we can remove the later 2 lines.